schema/elasticsearch/visibility/index_template.json

```diff
@@ -37,6 +37,7 @@
         "ExecutionStatus": {
           "type": "long"
         },
+
```

Comment:
I believe everything below this empty line (besides `Attr` of course) can be removed also because we don't use them in any search query.

Reply:
`HistoryLength` can be used after workflow close

plz keep the PR as is